docs/devTools/README.md

```diff
@@ -0,0 +1,62 @@
+# Dev tools
+
+Local development tools for deploying the [any.sender contracts](https://github.com/PISAresearch/contracts.any.sender) and hosting a mock any.sender instance. We also have a live ropsten API for when you're ready to take it to the next level :)
+
+## Deploying contracts locally
+
+To deploy the contracts import the `deploy` object from the `@any-sender/client` package.  The deploy object has an `contracts` function which accepts a provider and returns a promise of the contracts. Call this at some point in your tests before wrapping objects with `any.sender()`.
+
+```
+contracts(provider: Provider, options?: DeployOptions) => Promise<Contracts>
+```
+
+#### provider: JsonRpcProvider | AsyncSendable | string
+Can be an ethers `JsonRpcProvider`, `AsyncSendable` or a json rpc url as a `string`.
```

Comment:
is this different in web3? 

is it worth saying there is an etherjs dependency?

Reply:
We dont currrently support web3 - but you can do `deploy.contracts(ganache.provider())` without ethers. That's the asyncsendable thing. But since the `any.sender` object is ethers dependent I thought it best not to spell out all the web3 options until we support it accross the board